tornjak-frontend/src/components/tornjak-helper.tsx

```diff
@@ -26,6 +26,19 @@ class TornjakHelper extends Component<TornjakHelperProp, TornjakHelperState> {
     };
   }
 
+  // checkRolesAdminUser takes in userRoles
+  // return whether the user is granted admin role or not
+  checkRolesAdminUser(userRoles: string[]) {
+    const adminString = "admin";
+    const isAdmin = userRoles.find(element => {
+      if (element.toLowerCase().includes(adminString.toLowerCase())) {
+        return true;
+      }
+      return false;
```

Comment:
we can shorten the if statement to `return element.toLowerCase...`

Reply:
done